module_1/exercises.py

```diff
@@ -7,31 +7,55 @@
 
 """
 print(10 % 3)
+# 1
 
 print(5//2)
+#2
 
 print(5/3)
+#2
```

Comment:
correct answer is 1.6666666666666667. The point is, this is not a floor division